MetaMask/metamask-extension
# Show account name when qr scan in send screen

## Explanation



Currently, when I scan a QR code for sending a tx, I see that address is not resolved neither to Contact Name (if that address was a contact) nor to Account Name (if address is inside the Wallet)

With this PR it will show the account nickname form the address book or account name.

## More Information


 Fixes #15110 

## Screenshots/Screencaps



### Before



<img width="681" alt="image" src="https://user-images.githubusercontent.com/97883527/195819241-5304e5e3-6f54-460d-837c-c5585724072d.png">


### After



https://user-images.githubusercontent.com/97883527/195820028-2b870cd4-8fdb-4773-9053-b4f534c71b76.mov



## Manual Testing Steps



Open MM in 2 different browsers
Import Account 1 from MM1 into MM2
Open Account 1 details to expose QR
On MM2 click Send
Click QR code
Scan QR code from Account 1 -- see that Account 1 name is displayed

## Pre-Merge Checklist


### + If there are functional changes:
